Sources/UtilityBeltUITesting_XCTestSupport/Extensions/XCUIApplication+LaunchArguments.swift

```diff
@@ -0,0 +1,28 @@
+// Copyright © 2021 SpotHero, Inc. All rights reserved.
+
+#if canImport(XCTest)
+
+    import UtilityBeltUITesting
+    import XCTest
+
+    public extension XCUIApplication {
+        /// Convenience method for merging new launch arguments into the launch arguments array.
+        /// - Parameter objects: An array of launch argument objects that will be adding into the launch arguments.
+        func updateLaunchArguments(_ arguments: [LaunchArgument]) {
+            arguments.forEach { argument in
+                // Remove the argument in the case that it already exists.
+                self.removeLaunchArgument(argument)
```

Comment:
What about doing one of the options below? Then you might not even need the `removeLaunchArgument` function.
```swift
guard self.launchArguments.contains(argument.rawValue) else { 
     continue
}
```
or would this work ? 
```swift
self.launchArguments += arguments.map{ $0.rawValue }.filter { !self.launchArguments.contains($0) }
```

Reply:
I thought separating out an argument removal function might be useful, but I think it would be better to simplify using the second option you presented.